internal/pkg/manifest/job.go

```diff
@@ -0,0 +1,105 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package manifest provides functionality to create Manifest files.
+package manifest
```

Comment:
A general question: is it possible for `job` to reuse `svc.go` by renaming it to be `workload.go`? And then add a `job.go` to include only `job` specific parts?

Reply:
yeah, I'm sort of incorporating elements of `svc` into the job code so I can easily rename `svc` to `workload`